src/compile/mark/mark.ts

```diff
@@ -83,6 +85,117 @@ function parsePathMark(model: UnitModel) {
   }
 }
 
+const STACK_GROUP_PREFIX = 'stack_group_';
+
+function getStackGroups(model: UnitModel) {
```

Comment:
Can you add a comment to explain what the logic is here and that we need this code to support rounded corners for stacked bars?